plugin-animation/src/main/java/com/mapbox/maps/plugin/animation/CameraTransform.kt

```diff
@@ -62,4 +63,35 @@ internal object CameraTransform {
   }
 
   fun calculateScaleBy(amount: Double, currentZoom: Double) = log2(amount) + currentZoom
+
+  /**
+   * Extension function to wrap coordinate to one world copy.
+   */
+  fun Point.wrapCoordinate(): Point {
+    val lng = wrap(
+      value = this.longitude(),
+      min = -LONGITUDE_MAX.toDouble(),
+      max = LONGITUDE_MAX.toDouble()
+    )
+    return if (lng.isNaN()) Point.fromLngLat(this.longitude(), this.latitude())
+    else Point.fromLngLat(lng, this.latitude())
+  }
+
+  // Wrap value to the given range (including min, excluding max).
+  @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+  internal fun wrap(value: Double, min: Double, max: Double): Double {
+    if (value.ge(min) && value.less(max)) {
+      return value
+    } else if (value.eq(max)) {
+      return min
+    }
+    val delta = max - min
+    val wrapped = min + ((value - min) % delta)
+    return if (value < min) wrapped + delta else wrapped
+  }
+
+  private const val PRECISION = 1e-6
+  private fun Double.eq(other: Double) = abs(this - other) < PRECISION
+  private fun Double.ge(other: Double) = this > other || this.eq(other)
```

Comment:
should it be also with eps?

```suggestion
  private fun Double.ge(other: Double) = this + PRECISION > other || this.eq(other)
```

or 

```suggestion
  private fun Double.ge(other: Double) = (this - other) > PRECISION || this.eq(other)
```

same question for `less` function